src/main/fc/config.c

```diff
@@ -110,10 +110,22 @@ pidProfile_t *currentPidProfile;
 #define RX_SPI_DEFAULT_PROTOCOL 0
 #endif
 
+#ifdef BOARD_HAS_VOLTAGE_DIVIDER
+#define DEFAULT_VOLTAGE_METER_FEATURES FEATURE_VBAT
+#else
+#define DEFAULT_VOLTAGE_METER_FEATURES 0
+#endif
+#ifdef BOARD_HAS_CURRENT_SENSOR
```

Comment:
Rather than `#define`ing `BOARD_HAS_CURRENT_SENSOR` in `target.h` I think it's better to define `DEFAULT_CURRENT_METER_FEATURES` in `target.h` and then have here:
```
#ifndef DEFAULT_CURRENT_METER_FEATURES
#define DEFAULT_CURRENT_METER_FEATURES 0
#endif
```